packages/FormGroup/src/FormGroup.stories.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+
+import Section from '@components/section';
+import readme from '../README.md';
+
+import FormGroup from './FormGroup';
+import Input from '@igloo-ui/input';
+import Select, { SelectOption } from '@igloo-ui/select';
+import Textarea from '@igloo-ui/textarea';
+
+export default {
+  title: 'Components/FormGroup',
+  component: FormGroup,
+  parameters: {
+    description: readme,
+  },
+} as ComponentMeta<typeof FormGroup>;
+
+const Template: ComponentStory<typeof FormGroup> = (args) => (
+  <FormGroup {...args}>
+    <Input type="text" placeholder="John" error={true} />
```

Comment:
it's possible to link the `showError` with the input error ?

Reply:
good catch, this should be fixe dint he next build